.claude/skills/auto-verify/SKILL.md

```
@@ -44,11 +44,28 @@ npm run typecheck 2>/dev/null || npx tsc --noEmit
 npx eslint --max-warnings 0 <changed-files>
 ```
 
-5. Run git hygiene checks:
+5. Check docstring coverage on new/modified exported classes and public methods:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)